modules/Layout.js

```diff
@@ -186,8 +186,12 @@ Layout.prototype.render = function (l) {
         x+4,y+h-1,
         x,y+h-5,
         x,y+4
-      ], bg = l.selected?gfx.theme.bgH:gfx.theme.bg2;
-      gfx.setColor(bg).fillPoly(poly).setColor(l.selected ? gfx.theme.fgH : gfx.theme.fg2).drawPoly(poly);
+      ], bg = l.selected?gfx.theme.bgH:gfx.theme.bg2, 
+      btnborder = l.selected?gfx.theme.fgH:gfx.theme.fg2;
+    //add button color option
+    if(l.btnCol!==undefined) bg = l.btnCol;
+    if(l.btnBorder1!==undefined) btnborder = l.btnBorder;
```

Comment:
There's a type here: `btnBorder1` - so it seems this wasn't tested?

Also, if you were to set a color, it looks like the button is then unable to show when it is selected or not (which happens on Bangle.js 1 where there is no touchscreen and you must select using buttons)? It might be better to just set the colors to the defaults if the button is selected, but then allow them to be customised when it's not selected.

... also, do we want `l.btnCol`? It might be better just to use the pre-existing `l.bgCol` value which isn't used here anyway?